packages/Switches/src/index.tsx

```diff
@@ -0,0 +1,136 @@
+import withUtils from "@blaze-react/utils";
+import React, { FunctionComponent, useState } from "react";
+
+interface IOptions {
+  checked: boolean;
+  value: string;
+  disabled: boolean;
+  required: boolean;
+  label: string;
+  id?: string;
+}
+interface ISwitchesProps {
+  labelPosition?: string;
```

Comment:
label position would be top, left, bottom right ? in this case type it as this options please